tox.ini

```diff
@@ -10,7 +10,12 @@ commands =
     coverage combine
     coverage report
     coverage xml
-passenv = DATABASE_URL PORT aws_access_key_id aws_secret_access_key
+passenv = 
+    DATABASE_URL PORT 
+    aws_access_key_id 
+    aws_secret_access_key 
+    is_aws_mturk_requestor 
```

Comment:
Tests now assume the account linked to the configured AWS credentials is a requester account.